.github/workflows/optimize-ci.yml

```diff
@@ -24,6 +39,17 @@ permissions:
   pull-requests: write
 
 jobs:
+  detect-changes:
```

Comment:
I decided to bring here the action I've created in [this PR](https://github.com/camunda/camunda-optimize/pull/13663) in `camunda/camunda-optimize` repo